deploy/examples/cluster-on-pvc.yaml

```diff
@@ -131,7 +131,7 @@ spec:
                 requests:
                   storage: 10Gi
               # IMPORTANT: Change the storage class depending on your environment
-              storageClassName: gp2
+              storageClassName: gp3-csi
```

Comment:
Why gp3?

Reply:
I see two storageclasses  so I used gp2-csi for mons and gp3-csi for osds. I can change this to gp2-csi if you prefer

Reply:
That's in an openshift cluster? Maybe we just revert this file, until we get upstream feedback on changing it.